bob/bob_test.rb

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.